frontend/src/components/pages/CampOverview/UserSelect.tsx

```diff
@@ -0,0 +1,53 @@
+import { Text, VStack } from "@chakra-ui/react";
+import { Select, GroupBase, MultiValue } from "chakra-react-select";
+import React from "react";
+import { UserSelectOption } from "../../../types/UserTypes";
+
+// In menu, show user's name and email
+// In control, show user's name only
+const formatOptionLabel = (user: UserSelectOption, { context }: any) => {
+  if (context === "menu") {
+    return (
+      <>
+        <VStack alignItems="left" width="100%">
+          <Text textStyle="bodyRegular">{user.label}</Text>
+          <Text textStyle="xSmallRegular">{user.email}</Text>
+        </VStack>
+      </>
+    );
```

Comment:
nit: I don't think we need another component wrapping this

```suggestion
      <VStack alignItems="left" width="100%">
        <Text textStyle="bodyRegular">{user.label}</Text>
        <Text textStyle="xSmallRegular">{user.email}</Text>
      </VStack>
    );
```

Reply:
nvm i changed it!